### DISCUSSION TITLE
Updated to latest version 3.1.4 from 1.0.3 resulting of which DrawText have breaking changes

### DISCUSSION THREAD

**Discussion by tauqeer15** [SixLabors/ImageSharp#2774] [Q&A]
This is the code snippet which has error
internal virtual void DrawStringWithAlignment(string text, Font font, Brush brush, PointF location, TextOptions textGraphicsOptions)
{
    ImageSharpGraphics.Mutate(x => x.DrawText(textGraphicsOptions, text, font, brush, location));
}
It is not accepting TextOptions earlier it was accepting TextGraphicOptions in previous version

**Comment by JimBobSquarePants** [comment]
You jumped two major versions, and you didn't expect breaking changes?

I'm gonna go out on a limb here and guess you haven't looked at the release notes or API docs given that you added this discussion to the wrong repository.  

https://docs.sixlabors.com/api/ImageSharp.Drawing/SixLabors.ImageSharp.Drawing.Processing.DrawTextExtensions.html?q=drawtext
